project/src/main/webapp/script.js

```diff
@@ -181,6 +183,53 @@ function getSize(frequency,max,min) {
   return size;
 }
 
+/** Assigns and returns a color based on the sentiment value. */
+function getSentimentColor(sentimentValue) {
```

Comment:
Instead of doing a bunch of if/elses here, could we do some magic to generate the color from the sentimentValue? There are a ton of different ways to programmatically do color interpolation, see: https://graphicdesign.stackexchange.com/questions/83866/generating-a-series-of-colors-between-two-colors